### DISCUSSION TITLE
conflict with MCPWM

### DISCUSSION THREAD

**Discussion by Jeronimo59** [pschatzmann/arduino-audio-tools#1572] [Q&A] [answered]
Hello,

I am trying to combine PWM Audio playing and [motor control using MCPWM](https://docs.espressif.com/projects/esp-idf/en/v4.4/esp32/api-reference/peripherals/mcpwm.html#operatehttps://docs.espressif.com/projects/esp-idf/en/v4.4/esp32/api-reference/peripherals/mcpwm.html#operate) using a [Xiao ESP32-S3 Sense](https://wiki.seeedstudio.com/xiao_esp32s3_getting_started/).
There seems to be interferences with MCPWM_UNIT_0 as well as MCPWM_UNIT_1. 
Can you confirm ? Any workaround ?
Thanks,

**Comment by pschatzmann** [comment]
I don't know. I never used this...

**Reply by Jeronimo59** [reply, marked as answer]
Thanks. I found the issue was because the SPI port used by SD card interfered with the GPIO pin I used for PWM... 

Nothing to do with your great library ;-).
